documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -444,6 +444,12 @@ C++ Standard API                      libstdc++  libc++     MSVC
 ``std::push_heap``                    Tested     Tested     Tested
 ------------------------------------- ---------- ---------- ----------
 ``std::pop_heap``                     Tested     Tested     Tested
+------------------------------------ ---------- ---------- ----------
+``std::generate``                     Tested     Tested     Tested
```

Comment:
Please move std::generate row to upper for alphabetical order as before

Reply:
There is no alphabetical order in this file.